conference.js

```diff
@@ -20,7 +20,11 @@ import analytics from './modules/analytics/analytics';
 
 import EventEmitter from "events";
 
-import { conferenceFailed } from './react/features/base/conference';
+import {
+    CONFERENCE_JOINED,
```

Comment:
Generally, we'd like to dispatch actions using action creators (i.e. functions), not composing actions (i.e. objects) everywhere we dispatch the action. I'll take care of this one.